x/logic/interpreter/registry.go

```diff
@@ -112,6 +112,7 @@ var Registry = map[string]RegistryEntry{
 	"block_height/1":            {predicate.BlockHeight, 1},
 	"block_time/1":              {predicate.BlockTime, 1},
 	"bank_balances/2":           {predicate.BankBalances, 1},
+	"bank_spendable_coins/2":    {predicate.BankSpendableCoins, 1},
```

Comment:
I would have called it `bank_spendable_balances` to be consistent with `bank_balances` and to match the [bank module API](https://buf.build/cosmos/cosmos-sdk/docs/44fbb0df9cea049d588e76bf930177d777552cf3:cosmos.bank.v1beta1#cosmos.bank.v1beta1.Query.SpendableBalances).

This is an open question we can discuss it, I'm interested in your point of view :)

Reply:
Good question !! 😅 We have followed the [`BankKeeper` interface](https://docs.cosmos.network/v0.47/modules/bank#viewkeeper) naming that name `GetAllBalances` for the `bank_balances` and `SpendableCoins` or `LockedCoins` for other balances. But It's seems that the API doesn't follow this same naming and I'd be tempted to follow API naming since it seems more consistent to me. 
So I think you're right and we can rename `bank_spendable_balances/2`.

Reply:
Indeed I would have expected the keeper and the gRPC APIs to be aligned 😄 . @ccamel Any thoughts ?

Reply:
This is a good point. Agree with you. I think it's better to align with the visible surface of the service, hence the gRPC.

But I would propose to address the refactoring of predicate names + doc in another alignment PR, if you agree.